.github/workflows/update-eventuals-tutorial.yml

```diff
@@ -36,9 +36,14 @@ jobs:
           path: eventuals-tutorial
 
       - name: Update WORKSPACE.bazel
+        # Since the `buildifier` binary used by `update-eventuals-git-repository.sh`
```

Comment:
I think because you're using `run: |` you can move this entire comment directly above the the `eval` line so that it's easier to understand!